app/src/main/java/org/p2p/wallet/debug/settings/DebugSettingsInteractor.kt

```diff
@@ -20,4 +22,8 @@ class DebugSettingsInteractor(
         metadataInteractor.updateMetadata(metadata.copy(strigaMetadata = null))
         strigaStorageContract.clear()
     }
+
+    fun resetUserCountry() {
+        settingsInteractor.userCountryCode = null
```

Comment:
userCountryCode  -> currentUserCountry?

Reply:
I thought same, but since we don't have other names of country except "current", I'm not sure if it makes sense